test/flows/student_finance_calculator_flow_test.rb

```diff
@@ -0,0 +1,519 @@
+        assert_next_node :are_you_a_doctor_or_dentist?, for_response: "dental-medical-healthcare"
+      end
+
+      should "have a next node of outcome_uk_part_time_students for any response" do
+        add_responses what_type_of_student_are_you?: "uk-part-time",
+                      how_much_are_your_tuition_fees_per_year?: "6935",
+                      where_will_you_live_while_studying?: "at-home",
+                      whats_your_household_income?: "50,000",
+                      how_many_credits_will_you_study?: "10",
+                      how_many_credits_does_a_full_time_student_study?: "11",
+                      do_any_of_the_following_apply_all_uk_students?: "has-disability"
+        assert_next_node :outcome_uk_part_time_students, for_response: "dental-medical-healthcare"
+      end
+    end
+  end
+
+  context "question: are_you_a_doctor_or_dentist?," do
+    setup do
+      testing_node :are_you_a_doctor_or_dentist?
+      add_responses when_does_your_course_start?: "2020-2021",
+                    what_type_of_student_are_you?: "uk-full-time",
+                    how_much_are_your_tuition_fees_per_year?: "9250",
+                    where_will_you_live_while_studying?: "at-home",
+                    whats_your_household_income?: "50,000",
+                    do_any_of_the_following_apply_uk_full_time_students_only?: "children-under-17",
+                    what_course_are_you_studying?: "dental-medical-healthcare"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of outcome_uk_full_time_students for a no response" do
+        assert_next_node :outcome_uk_full_time_students, for_response: "no"
+      end
+
+      should "have a next node of outcome_uk_full_time_dental_medical_students for a yes response" do
+        assert_next_node :outcome_uk_full_time_dental_medical_students, for_response: "yes"
+      end
+    end
+  end
+
+  context "outcome: outcome_uk_full_time_students" do
+    setup do
+      testing_node :outcome_uk_full_time_students
+      add_responses when_does_your_course_start?: "2020-2021",
+                    what_type_of_student_are_you?: "uk-full-time",
+                    how_much_are_your_tuition_fees_per_year?: "9250",
+                    where_will_you_live_while_studying?: "at-home",
+                    whats_your_household_income?: "50,000",
+                    do_any_of_the_following_apply_uk_full_time_students_only?: "children-under-17",
+                    what_course_are_you_studying?: "teacher-training"
+    end
+
+    should "render text for when the student is eligible for extra grants and allowances" do
+      assert_rendered_outcome text: "Depending on your income and circumstances"
+    end
+
+    should "render text for when the student is not eligible for extra grants and allowances" do
```

Comment:
Super minor, can we make this the first test because it's the first thing in that outcome